HopsanGUI/Dialogs/ComponentPropertiesDialog3.cpp

```diff
@@ -1793,6 +1802,14 @@ void ParameterValueSelectionWidget::openValueEditDialog()
     }
 }
 
+void ParameterValueSelectionWidget::openFileBrowserDialog()
+{
+    QString filePath = QFileDialog::getOpenFileName(gpMainWindowWidget, "Select file", gpDesktopHandler->getDocumentsPath());
+    if(!filePath.isEmpty()) {
+        mpValueEdit->setText(filePath);
```

Comment:
Maybe we should open an additional dialog to let the user choose if the path should be absolute or relative to the model file. Or automatically make it relative to the model file automatically. I am not sure. If its absolute then if you "send" the model to someone else, paths may be wrong. But I think that relative paths should work. Provided that the same relative path is available on the other side.

Maybe make it relative automatically if the "data file" is on the same level deeper down the directory structure from where the model is located.